projects/packages/jetpack-mu-wpcom/src/features/holiday-snow/holiday-snow.scss

```diff
@@ -28,7 +28,7 @@ html {
 	@for $i from 0 to $snow_density {
 		$v: list.nth($snow_sizes, $i + 1);
 		$alpha: list.nth($snow_alphas, $i + 1);
-		$grad: $grad, radial-gradient($v + px $v + px at (round(list.nth($snow_positions, $i * 2 + 1) * ($grid_width - $v * 2)) + $v) + px (round(list.nth($snow_positions, $i * 2 + 2) * ($grid_width - $v * 2)) + $v) + px,
+		$grad: $grad, radial-gradient($v + px $v + px at (math.round(list.nth($snow_positions, $i * 2 + 1) * ($grid_width - $v * 2)) + $v) + px (math.round(list.nth($snow_positions, $i * 2 + 2) * ($grid_width - $v * 2)) + $v) + px,
```

Comment:
It seems `round()` is a native CSS function newer browsers. I suppose SASS would use its deprecated call rather than the native call.

Reply:
I don't know that Sass's `round()` is actually deprecated, but switching to the explicit `math.round()` is probably a good idea.

Really the change here was done by Sass's own migrator. 😀

Reply:
😅 I was looking in the logs but didn't see any deprecated notice for that, but assumed I was missing something.